docs/guides/cloud-setup-invited.mdx

```diff
@@ -0,0 +1,220 @@
+---
+title: Set up your IBM Cloud account - invited users
+description: Setup instructions for IBM Quantum on IBM Cloud to submit Qiskit SDK and Qiskit Runtime jobs for users with email invitations
+platform: cloud
+---
+
+{/* cspell:ignore urlendcoded, WQVY */}
+
+# Set up your IBM Cloud account - invited users
+
+If you do not have an invitation from an administrator to join an account, follow the steps in [Set up your IBM Cloud account](/docs/guides/cloud-setup) instead.
+
+You can run workloads on IBM&reg; quantum processing units (QPUs) by setting up an account on IBM Cloud&reg;. You will also need to [install and set up Qiskit and Qiskit Runtime](/docs/guides/install-qiskit#local).
+
+
+Your user account is associated with one or more [instances](./instances) that give access to IBM Quantum services. Additionally, a unique token is assigned to each account -- known as a *Cloud Resource Name* (CRN) -- allowing for IBM Quantum access from Qiskit.  For instructions to choose a specific instance, see [Connect to an instance](./instances#connect-instance).
+
+    <Admonition type="note">
+      The [IBM Quantum Platform dashboard](https://quantum.cloud.ibm.com) and the [Instances page](https://quantum.cloud.ibm.com/instances) list the instances that you can access.
+    </Admonition>
+
+<span id="cloud"></span>
+## Set up to use IBM Cloud
+
+### Before you begin
+
+1. Ensure you are working in an active Python environment with the [Qiskit SDK and Qiskit Runtime installed](/docs/guides/install-qiskit#local).
+
+1. Accept the email invitation.
+1. If you do not already have one, set up an [IBM Cloud account.](https://quantum.cloud.ibm.com/registration)
```

Comment:
accepting the email invitation you don't need to setup an IBM Cloud account, you're invited to one. You will need to log in with an IBMid or Google account. If you don't have one, you'll be guided through creating one during the join process. After logging in, you'll be asked to confirm your information and accept the invitation to join the account. 

After joining the account users will land in IBM Cloud, users will need navigate back to IBM Quantum Platform (quantum.cloud.ibm.com).